### PR TITLE
FAQ and Troubleshooting article

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
This PR is improving the current FAQ and Troubleshooting articles that we currently have.

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5890/

File: docs/FAQ/README.md [hunk 1]
```diff
@@ -37,6 +39,20 @@ import { Button } from 'wix-style-react';
 
 `import`ing a component directly from `dist/src` is strongly prohibited.
 
+### I'm not sure which component to use to implement the desired spec
+We recommend checking out our [cheatsheet](https://www.wix-style-react.com/?path=/story/introduction-cheatsheet--components-cheatsheet) page which includes a view of most of the library components inventory.
+Furthermore, it includes links to the components and ux stories. If you cannot find it there and you're unable to find it by the regular search, please contact us at #wix-style-ux or #wix-style-react channels.
+
+### The component I'm using does not look or behave as expected
+Firstly, make sure that you use the component as described within the examples and API description. Should the issue persists,
+please try to recreate it within storybook. Once you are able to recreate it, please contact #wix-style-react channel and provide us with the following information:
```
Comment (gendeld) [inline docs/FAQ/README.md:48]: Maybe add a link to the playground here

File: docs/usage/Troubleshooting.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
-# What should i do when i get in trouble?
+# What should I do when I get in trouble?
 
-When i find my self in times of trouble, this small readme comes to me.
```
Comment (gendeld) [inline docs/usage/Troubleshooting.md:3]: image

Comment (TalZa1810) [reply docs/usage/Troubleshooting.md:3]: LOL

File: docs/usage/Troubleshooting.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+1. Check whether there is an example that fits your scenario in the [storybook](https://www.wix-style-react.com).
+2. Check our [Cheetsheet](https://www.wix-style-react.com/?path=/story/introduction-cheatsheet--components-cheatsheet) which includes a view of most of the library components inventory and links to the relevant documentation pages.
+3. Use the [Playground](https://www.wix-style-react.com/?path=/story/introduction-playground--playground) to easily test the component abilities.
+4. You may find an answer to your question in our [Frequently Asked Questions](https://github.com/wix/wix-style-react/tree/master/docs/FAQ#frequently-asked-questions) article.
+5. Each component within the library is tested with our on drivers. This is a great way to document the proper usage of the component and drivers. You can learn from the tests on how the component should properly used / tested. Checkout the [sourcecode](https://github.com/wix/wix-style-react). Each component's tests files are listed under:
```
Comment (gendeld) [inline docs/usage/Troubleshooting.md:9]:
```suggestion
5. Each component within the library is tested with our own drivers. This is a great way to document the proper usage of the component and drivers. You can read the tests to learn how the component should be properly used and tested. Checkout the [sourcecode](https://github.com/wix/wix-style-react). Each component's tests files are listed under:
```

File: docs/usage/Troubleshooting.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+`src/ComponentName/test/...`.
+6. Create a new [Yoshi](https://github.com/wix/yoshi) project to serve as a clean environment to test the component - isolate all the noise you got from your own project.
+7. Strip down abilities until you get to the bottom of it. Saying we have for example the `<Table/>` component which does not work as we expect it to. All the above steps did not help us. The thing we would do is one of the two:
```
Comment (gendeld) [inline docs/usage/Troubleshooting.md:12]:
```suggestion
7. Strip down abilities until you get to the bottom of it. Let's say we have the `<Table/>` component which does not work as expected. All the above steps did not help us. We could try one of the following:
```

**Comment by gendeld** [comment]
A+ 🎖

File: docs/FAQ/README.md [hunk 5]
```diff
@@ -37,6 +39,20 @@ import { Button } from 'wix-style-react';
 
 `import`ing a component directly from `dist/src` is strongly prohibited.
 
+### I'm not sure which component to use to implement the desired spec
+We recommend checking out our [cheatsheet](https://www.wix-style-react.com/?path=/story/introduction-cheatsheet--components-cheatsheet) page which includes a view of most of the library components inventory.
+Furthermore, it includes links to the components and ux stories. If you cannot find it there and you're unable to find it by the regular search, please contact us at #wix-style-ux or #wix-style-react channels.
+
+### The component I'm using does not look or behave as expected
+Firstly, make sure that you use the component as described within the examples and API description. Should the issue persists,
```
Comment (argshook) [inline docs/FAQ/README.md:47]:
```suggestion
Firstly, make sure that you use the component as described within the examples and API description. Should the issue persist,
```

File: docs/FAQ/README.md (hunk 1 above)
Comment (argshook) [inline docs/FAQ/README.md:48]:
```suggestion
please try to recreate it in the [playground](https://www.wix-style-react.com/?path=/story/introduction-playground--playground). Once you are able to recreate it, please contact #wix-style-react channel and provide us with the following information:
```

File: docs/usage/Troubleshooting.md [hunk 6]
```diff
@@ -1,20 +1,19 @@
...
+1. Check whether there is an example that fits your scenario in the [storybook](https://www.wix-style-react.com).
+2. Check our [Cheetsheet](https://www.wix-style-react.com/?path=/story/introduction-cheatsheet--components-cheatsheet) which includes a view of most of the library components inventory and links to the relevant documentation pages.
+3. Use the [Playground](https://www.wix-style-react.com/?path=/story/introduction-playground--playground) to easily test the component abilities.
+4. You may find an answer to your question in our [Frequently Asked Questions](https://github.com/wix/wix-style-react/tree/master/docs/FAQ#frequently-asked-questions) article.
+5. Each component within the library is tested with our own drivers. This is a great way to document the proper usage of the component and drivers. You can read the tests to learn how the component should be properly used and tested. Checkout the [sourcecode](https://github.com/wix/wix-style-react). Each component's tests files are listed under:
```
Comment (argshook) [inline docs/usage/Troubleshooting.md:9]:
```suggestion
5. Each component within the library is tested with our own drivers. This is a great way to document the proper usage of the component and drivers. You can read the tests to learn how the component should be properly used and tested. Checkout the [source code](https://github.com/wix/wix-style-react). Each component's tests files are listed under:
```

File: docs/usage/Troubleshooting.md [hunk 7]
```diff
@@ -1,20 +1,19 @@
...
+`src/ComponentName/test/...`.
+6. Create a new [Yoshi](https://github.com/wix/yoshi) project to serve as a clean environment to test the component - isolate all the noise you got from your own project.
```
Comment (argshook) [inline docs/usage/Troubleshooting.md:11]: should probably remove this suggestion, as yoshi is not open source anymore

**Pull request merged**